### PR TITLE
Added endpoints for game paging and name search. Mongodb atlas integration

### PR DESCRIPTION
**Pull request by Konkord360**

### CODE REVIEW

**Review by m-borkowski — APPROVED** [review]
LGTM

File: src/main/java/com/polsl/gamelib/controller/GameController.java [hunk 1]
```diff
@@ -10,26 +16,95 @@
...
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
-import java.util.Optional;
 
 @Controller
 public class GameController {
+    final String REVIEWS_URI = "https://store.steampowered.com/api/appdetails?appids=%s";
+    final String DETAILS_URI = "https://store.steampowered.com/api/appdetails?appids=%s";
```
Comment (macholewka) [inline src/main/java/com/polsl/gamelib/controller/GameController.java:30]: Both uri's are the same. One should be enough

**Review by macholewka — APPROVED** [review]

**Comment by macholewka** [comment]
Don't forget to do rebase instead of merge!!!

File: src/main/java/com/polsl/gamelib/controller/GameController.java (hunk 1 above)
Comment (Konkord360) [reply src/main/java/com/polsl/gamelib/controller/GameController.java:30]: That's an error. Should be different.

**Pull request merged**